internal/capability/auditor.go

```diff
@@ -50,15 +50,20 @@ func (capAuditor *capAuditor) BuildWorkQueueByCatalog(catalogSource string, cata
 	}
 
 	// build workqueue as buffered channel based subscriptionData list size
-	capAuditor.WorkQueue = make(chan capAudit, len(s))
+	capAuditor.WorkQueue = make(chan CapAudit, len(s))
 	defer close(capAuditor.WorkQueue)
 
 	// add capAudits to the workqueue
 	for _, subscription := range s {
 
-		// load workqueue with capAudit
-		capAuditor.WorkQueue <- newCapAudit(c, subscription, auditPlan)
+		capAudit, err := newCapAudit(c, subscription, auditPlan)
+		if err != nil {
+			logger.Debugw("Couldn't build capAudit for subscription %s", "Err:", err)
```

Comment:
Could you change this to Debugf instead of Debugw. Debugw will not accept this format.

Reply:
done!